tests/image-stream/metadata/zip.spec.js

```diff
@@ -127,19 +126,18 @@ describe('ImageStream: Metadata ZIP', function() {
 
   describe('given an archive with a bmap file', function() {
 
-    const archive = path.join(ZIP_PATH, 'rpi-with-bmap.zip');
+    const archive = path.join(ZIP_PATH, 'etcher-test-with-bmap.zip');
 
     const bmap = [
       '<?xml version="1.0" ?>',
       '<bmap version="1.3">',
-      '    <ImageSize> 36864 </ImageSize>',
+      '    <ImageSize> 5242880 </ImageSize>',
       '    <BlockSize> 4096 </BlockSize>',
-      '    <BlocksCount> 9 </BlocksCount>',
-      '    <MappedBlocksCount> 4     </MappedBlocksCount>',
-      '    <BmapFileSHA1> d90f372215cbbef8801caca7b1dd7e587b2142cc </BmapFileSHA1>',
+      '    <BlocksCount> 1280 </BlocksCount>',
+      '    <MappedBlocksCount> 1280 </MappedBlocksCount>',
+      '    <BmapFileSHA1> cc6f077565c73a46198777b259c231875df4e709 </BmapFileSHA1>',
       '    <BlockMap>',
-      '        <Range sha1="193edb53bde599f58369f4e83a6c5d54b96819ce"> 0-1 </Range>',
-      '        <Range sha1="193edb53bde599f58369f4e83a6c5d54b96819ce"> 7-8 </Range>',
+      '        <Range sha1="7b7d6e1fc44ef224a8c57d3ec6ffc3717c428a14"> 0-1280 </Range>',
```

Comment:
Yeah, that's because I couldn't get Tizen's bmap-tool to function, it kept complaining about `ioctl` not returning something it wanted. Handcrafted it instead. The test is about the discovering & reading the metadata, not blockmapping – and since this is only a 5MB disk image, this wouldn't be a good test for blockmaps either way – so this doesn't really matter imo.